### PR TITLE
#9808 included logs on securitylogger

### PR DESCRIPTION
**Pull request by oswaldogallango**

### CODE REVIEW

File: src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java [hunk 1]
```diff
@@ -41,5 +49,36 @@ public Response getErrorResponse(final HttpServletRequest request,
...
         return ErrorResponseHelper.INSTANCE.getErrorResponse(status, locale, messageKey);
     }
+    
+    /**
+	 * Add a info message in the security log
+	 * @param dotCMSClass Class responsible for the message
+	 * @param message Message to be printed in the log
+	 */
+	public void securityLog(Class dotCMSClass, String message) {
+		SecurityLogger.logInfo(dotCMSClass, message);
```
Comment (freddyucv) [inline src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java]: why don't use the SecurityLogger directly in the Resource?

Comment (jdotcms) [inline src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java]: I have not any issue to do it on the Resource or Helper.

**Review by jdotcms — APPROVED** [review]

File: src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java [hunk 2]
```diff
@@ -41,5 +49,36 @@ public Response getErrorResponse(final HttpServletRequest request,
...
+	
+	/**
+	 * Get the translation of the message key in the specified locale, if the locale is null
+	 * then the message is translated into the default user language
+	 * @param locale Current user language
+	 * @param messageKey Message key to be translated
+	 * @param messageArguments (Optional) if the message require some argument
+	 * @return
+	 */
+	public String getFormattedMessage(Locale locale, String messageKey,Object... messageArguments){
```
Comment (oarrietadotcms) [inline src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java]: Spaces vs Tabs on this new code

File: src/com/dotcms/rest/api/v1/authentication/ResetPasswordResource.java [hunk 3]
```diff
@@ -76,7 +73,7 @@ public final Response resetPassword(@Context final HttpServletRequest request,
 
             jwtBean = this.jsonWebTokenService.parseToken(jwtToken);
             if (null == jwtBean) {
-
+            	this.authenticationHelper.securityLog(ResetPasswordResource.class,"Error reseting password. "+this.authenticationHelper.getFormattedMessage(null,"reset-password-token-expired"));
```
Comment (oarrietadotcms) [inline src/com/dotcms/rest/api/v1/authentication/ResetPasswordResource.java]: I think we have too many characters on the same line

File: src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java (hunk 1 above)
Comment (oswaldogallango) [inline src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java]: Done, changed to use directly the security logger

File: src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java (hunk 2 above)
Comment (oswaldogallango) [inline src/com/dotcms/rest/api/v1/authentication/AuthenticationHelper.java]: Done.

**Pull request merged**